sematic/ee/LICENSE

```diff
@@ -0,0 +1,36 @@
+The Sematic Enterprise license (the "Enterprise License")
+Copyright (c) 2023 - present Sematic Inc.
+
+With regard to the Sematic Software:
+
+This software and associated documentation files (the "Software") may only be
+used in production, if you (and any entity that you represent) have agreed to,
+and are in compliance with, the Sematic Enterprise Terms of Service, available
+via email (emmanuel@sematic.dev) (the "Enterprise Terms"), or other
```

Comment:
Would be more professional to put in a mailing alias like `sales@sematic.dev` or `enterprise@sematic.dev` in here.